.github/workflows/build-installers.yml

```diff
@@ -0,0 +1,38 @@
+name: Build-Installers
+
+on:
+  push:
+    branches: [ master ]

```

Comment:
We will likely want these on `release` branch, too? cc: @mjcheetham

Reply:
Yes please 👍